frontend/src/Buildings/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,338 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
you should shallow render these complex components:
- [shallow renderer example](https://stackoverflow.com/a/42401039/2163037)